### DISCUSSION TITLE
Get value from 200 questions

### DISCUSSION THREAD

**Discussion by JackRustig** [PHPMailer/PHPMailer#2857] [Q&A] [answered]
Hello, 

i have a specific type of form that contains up to 200 questions , majority in the form of radiobuttons. 
How can i grab all the values submited without having to manually naming them in my php?
        $mail->addAddress($to);
        $mail->addReplyTo($email, $name);
        $mail->Subject = 'Contact form: ' . $subject;
        $mail->Body = "Contact form submission\n\n" . $query;
        **$mail-> GET ALL VALUES FROM RADIOBUTTONS with their unique names??**

**Comment by Synchro** [comment, marked as answer]
This isn't really a PHPMailer question. There are many ways of doing that, but you could simply put everything in the request into the message like this:

```php
foreach ($_REQUEST as $key => $value) {
    $mail->Body .= "$key = $value\n";
}
```

Bear in mind that the way that checkboxes work in HTML is that if they are unchecked, they will not be included in the form submission, so you have to infer that they are unchecked by comparing them to a list of expected names.
